tools/segmetrics/segmetrics.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="ip_segmetrics" name="Compute image segmentation and object detection performance measures" version="1.4.0-2" profile="20.05">
+<tool id="ip_segmetrics" name="Compute image segmentation and object detection performance measures" version="1.4.0+galaxy3" profile="20.05">

```

Comment:
It would be nice to use our version recommendations at some point: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=version#tool-versions

This will enable out bot to create udpate PRs.

Reply:
Thanks for pointing that out! That’s really good to know, I missed that part of the docs.